### PR TITLE
Alure pkgbuild

### PR DESCRIPTION
**Pull request by fundies**

### CODE REVIEW

**Comment by Alexpux** [comment]
Format PKGBUILD. Replace all TABs with spaces. In depends - one pakage per line.

File: mingw-w64-alure/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  cd "${srcdir}/${_realname}-${pkgver}"
+  patch -p1 -i "${srcdir}/build.patch"
+}
+
+build() {
+  [[ -d "${srcdir}/build-${MINGW_CHOST}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}"
+  mkdir -p "${srcdir}/build-${MINGW_CHOST}"
+  cd "${srcdir}/build-${MINGW_CHOST}"
+
+  ${MINGW_PREFIX}/bin/cmake ../${_realname}-${pkgver} \
```
Comment (Alexpux) [inline mingw-w64-alure/PKGBUILD]: You need place MSYS2_ARG_CONV_EXCL line before cmake. See other cmake based packages.
Also place source folder to next line.

File: mingw-w64-alure/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Maintainer: Greg Williamson <cheeseboy16@gmail.com>
+
+_realname=alure
+pkgbase="mingw-w64-${_realname}"
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.2
+pkgrel=5
+pkgdesc='Utility library to help manage common tasks with OpenAL applications.'
```
Comment (Alexpux) [inline mingw-w64-alure/PKGBUILD]: Remove ````.```` from description and add ````(mingw-w64)```` to the end

**Comment by fundies** [comment]
well that broke  :/

**Comment by fundies** [comment]
ok done everything youve asked and squashed

**Pull request merged**